ClientApp/utils/context/AuthContext.tsx

```diff
@@ -110,13 +110,14 @@ export const AuthProvider: React.FC<{ children: ReactNode }> = ({ children }) =>
   const login = async (identifier: string, password: string): Promise<{ success: boolean; data:{identifier: string}, error?: string }> => {
     const accessToken = await getAccessToken();
     const refreshToken = await getRefreshToken();
+    console.log("Checkpoint 1")
```

Comment:
I dont know if these are meant to be here butwe can remove them in another implementation!

Reply:
yes, good catch!. That was me when I refactoring/debugging. If you can remove it @yousfino, thank you!

Reply:
resolved !